Samples/london-no2.py

```diff
@@ -12,5 +16,5 @@
 
 dataset = Dataset(subjects=[subjects], datasources=[datasources], fields=[lvf])
 recipe = Recipe(dataset=dataset)
-recipe.build_recipe(output_location='Desktop/london-no2.json')
-recipe.run_recipe(tombolo_path='Desktop/UptodateProject/TomboloDigitalConnector', output_path='Desktop/london-no2.geojson')
+recipe.build_recipe(output_location=recipe_output_location)
+recipe.run_recipe(tombolo_path=tombolo_path, output_path=model_output)
```

Comment:
every recipe i run builds succesfully and produces a result  but the logs end with 

An exception has occurred, use %tb to see the full traceback.

SystemExit: 1

why'?